lib/src/generator/templates.runtime_renderers.dart

```diff
@@ -5,6 +5,7 @@
 
 // ignore_for_file: camel_case_types, deprecated_member_use_from_same_package
 // ignore_for_file: unused_import
+// @dart=2.9
```

Comment:
I assume this language comment is ok here (after the ignore comments).

Reply:
It seems to be!  If it wasn't, it would definitely fail tests at this point.